src/main/java/com/grud/ConnectionDb.java

```diff
@@ -0,0 +1,21 @@
+package com.grud;
+
+import lombok.SneakyThrows;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.Properties;
+
+public class ConnectionDb {
```

Comment:
As it is a util class. Better to make it final and create a private constructor to prevent creation object of this class.
And you can put this class into package com.gruf.util for example.